src/main/java/org/jsoup/parser/TokeniserState.java

```diff
@@ -565,8 +565,8 @@ void read(Tokeniser t, CharacterReader r) {
                     t.transition(SelfClosingStartTag);
                     break;
                 case '<': // NOTE: out of spec, but clear (spec has this as a part of the attribute name)
-                    t.error(this);
                     r.unconsume();
+                    t.error(this);
```

Comment:
Should this also be changed in the `nullChar` case 6 lines below this line?

Reply:
You're right. I'll fix this.